docs/development/test-vectors.rst

```diff
@@ -546,6 +546,12 @@ Custom X.509 Vectors
   This is an invalid certificate per :rfc:`5280` 4.2.1.12.
 * ``malformed-san.pem`` - A certificate with a malformed SAN.
 * ``malformed-ian.pem`` - A certificate with a malformed IAN.
+* ``admissions_extension_cert_synthetic.pem`` - A certificate containing
+  the ``Admissions`` extension with synthetic data,
+  signed by ``x509/custom/ca/rsa_ca.pem`` CA.
```

Comment:
"synthetic data" is a bit unclear, can you describe the structure of this cert, what makes it interesting/what is it for testing.